pkg/generated/protobuf/rekor_service.pb.go

```diff
@@ -68,40 +74,348 @@ func (x *StringMessage) ProtoReflect() protoreflect.Message {
 	return mi.MessageOf(x)
 }
 
-// Deprecated: Use StringMessage.ProtoReflect.Descriptor instead.
-func (*StringMessage) Descriptor() ([]byte, []int) {
+// Deprecated: Use CreateEntryRequest.ProtoReflect.Descriptor instead.
```

Comment:
I know this is generated but why are we using something that's already deprecated?

Reply:
I dunno, this is what the generator gives us.